.github/ISSUE_TEMPLATE/04-feature-request.yml

```diff
@@ -19,6 +19,12 @@ body:
         - [Guide on creating issues + FAQ + Commonly Known Issues](https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/main/CREATING_ISSUES.md) - If you would like to create a GitHub issue, please read our guide first. It contains useful links, FAQ, and commonly known issues with solutions that may already solve your case.
         - [Provider Roadmap](https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/main/ROADMAP.md)
 
+  - id: company-name
+    type: input
+    attributes:
+      label: Company Name
+      description: Optional field. Providing this information allows us to communicate faster which may lead to faster issue resolution.
```

Comment:
Don't know if in all templates we should refer to faster resolution of "issue" in e.g. feature request. But, other than that 👍.

Reply:
feature request is also a GH issue, I think it's fine

Reply:
Why not? In this case, resolved issue == implemented feature.